src/css/GameScreen.css

```diff
@@ -956,48 +956,36 @@
 /*Animation deck to hand first deal*/
 .card-back {
     opacity: 0.9;
-    position: absolute;
-    transform: rotateY(180deg);
-       
+    position: absolute;     
 }
 
 .player-top .card-back {
     transform: rotateX(180deg);
 }
 
-.animated-card-0 {
-    animation: deckToHand-0 3s ease-in;
+.animated-card {
     animation-direction:alternate;
```

Comment:
you must add a space between property and value in the css rule.

Reply:
thanks!